src/storage/pipes/storageUrl.pipe.ts

```diff
@@ -0,0 +1,13 @@
+import { Pipe } from '@angular/core'
+import { AngularFireStorage } from '../storage';
+
+/** to be used with in combination with | async */
+@Pipe({
+  name: 'ngfbStorageUrl'
```

Comment:
Perhaps we could call this `downloadURL`, `getDownloadURL`, `storageDownloadURL` or something to that effect, @davideast any thoughts?

Could we make it so this was an async pipe itself so it didn't have to be combined with async?  So the developer could just do `'someref.jpeg' | storageDownloadURL`... or maybe then we call it `asyncDownloadURL`?

Reply:
My experience and recommendation is to avoid being too generic as to avoid name collisions as this type of functionality is common to convert a string to a url.

Also, if you removed the need for the async pipe, appending a dollar sign `ngfbStorageUrl$` is a standard I’ve read to be clear that an in template variable is or is going through an observable process. It’s a practice to make clear something async is at play. I actually recommend against removing the need for the async pipe OR offer both (one pipe with a trailing $ sign and one without)

Reply:
Both would be cool, but we could scope as a separate PR if you felt so inclined.

How about `getFirebaseDownloadURL` and `firebaseDownloadURL$` for naming then? That way it's super clear.

Reply:
I like that a lot. Great collaboration. I’d like to have this code in my project, how can I help move this along?

I can make the changes but perhaps @davideast would like throw an opinion in the hat?

Thank you very kindly

Reply:
Ps, I don’t know how to have a pipe that has to be async not require the async pipe.

The end all be all maybe that the async pipe is actually a requirement

Reply:
Sorry to “spam”

Here is dollar sign suffix docs:
https://angular.io/guide/rx-library#naming-conventions-for-observables